### PR TITLE
Docs for specifying base image:tag

### LINKED ISSUE #468: Document ContainerBaseImage and ContainerBaseTag
Now that https://github.com/dotnet/tye/pull/447/ is merged, we should add documentation for that in the next tye release.

### PR DESCRIPTION
**Pull request by spboyer**
Fixes #468

### CODE REVIEW

File: docs/reference/local_development.md [hunk 1]
```diff
@@ -109,6 +109,30 @@ Tye will build all .NET projects (unless `build: false` has been specified) befo
 
 Tye will use the Docker cli to pull any needed container images before launching any services. This is done as a separate step to avoid failures when the network is very slow or an image is very large. 
 
+### Determining Docker base images and tags
+
+Tye will use the TFM of the project to set the base image and tags of the container images to use when creating the associated Dockerfile.
+
+Should you choose to use a custom base image and/or tag or variant you can do so by soecifying the base image and tag in a `<PropertyGroup>` in any of the projects.
```
Comment (jkotalik) [inline docs/reference/local_development.md:116]:
```suggestion
Should you choose to use a custom base image and/or tag or variant you can do so by specifying the base image and tag in a `<PropertyGroup>` in any of the projects.
```

File: docs/reference/local_development.md [hunk 2]
```diff
@@ -109,6 +109,30 @@ Tye will build all .NET projects (unless `build: false` has been specified) befo
...
+
+Example - custom image
+```xml
+<PropertyGroup>
+   <ContainerBaseImage>myrepository/dotnetbaseimage</ContainerBaseImage>
```
Comment (jkotalik) [inline docs/reference/local_development.md:121]:
```suggestion
   <ContainerBaseImage>myregistry/dotnetbaseimage</ContainerBaseImage>
```

File: docs/reference/local_development.md [hunk 3]
```diff
@@ -109,6 +109,30 @@ Tye will build all .NET projects (unless `build: false` has been specified) befo
...
+
+Example - Debian buster tag
+```xml
+<PropertyGroup>
+   <ContainerBaseImage>mcr.microsoft.com/dotnet/core/sdk</ContainerBaseImage>
+   <ContainerBaseTag>3.1-buster</ContainerBaseTag>
+</PropertyGroup>
+```
+
+_* Note only public repositories are supported._
```
Comment (jkotalik) [inline docs/reference/local_development.md:134]: Wouldn't it work for private repositories if you are logged in, ex if you are logged into ACR, wouldn't this work?

Comment (spboyer) [reply docs/reference/local_development.md:134]: ACK - removed comments. Didn't think through that completely.

**Pull request merged**